src/Pmi/Console/Command/MetricsCommand.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace Pmi\Console\Command;
+
+use Pmi\Application\HpoApplication;
+use Pmi\Drc\RdrMetrics;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Formatter\OutputFormatter;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Metrics Command
+ */
+class MetricsCommand extends Command
+{
+    // Allowed stratfications in RDR
+    const STRATIFICATIONS = ['TOTAL', 'ENROLLMENT_STATUS'];
```

Comment:
While deploying the code we found out that one of the task in our deploy script is using php 5.5 version where array class constants are not supported so it's throwing an error.